modules/local/merge_predictions.nf

```diff
@@ -0,0 +1,30 @@
+process MERGE_PREDICTIONS {
+    label "process_high_memory"
+    
+    conda (params.enable_conda ? "bioconda::csvtk=0.23.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/csvtk:0.23.0--h9ee0642_0' :
+        'quay.io/biocontainers/csvtk:0.23.0--h9ee0642_0' }"
+
+    input:
+    path predictions
+    path prediction_warnings
+
+    output:
+    path "predictions.tsv.gz",          emit: ch_predictions
+    path "prediction_warnings.log"
+    path "versions.yml",                emit: versions
+
+    script:
+    def single = predictions instanceof Path ? 1 : predictions.size()
```

Comment:
I have never seen that expression `predictions instanceof Path`, I expected here something like `predictions.instanceof(Path)` (but no guarantee). Just asking.

Reply:
I tested it and it works as expected.

Reply:
Alright!